include/aos/common/config.hpp

```diff
@@ -428,6 +435,34 @@
 #define AOS_CONFIG_TYPES_VERSION_LEN 32
 #endif
 
+/**
+ * Max subnet length.
+ */
+#ifndef AOS_CONFIG_TYPES_NETWORKMANAGER_SUBNET_LEN
```

Comment:
Remove `NETWORKMANAGER` all these consts belong to types.hpp

Reply:
Done